doc/motion_planning_api/src/motion_planning_api_tutorial.cpp

```diff
@@ -34,50 +34,57 @@
 
 /* Author: Sachin Chitta, Michael Lautman */
 
-#include <pluginlib/class_loader.h>
-#include <ros/ros.h>
+#include <pluginlib/class_loader.hpp>
 
 // MoveIt
 #include <moveit/robot_model_loader/robot_model_loader.h>
 #include <moveit/planning_interface/planning_interface.h>
 #include <moveit/planning_scene/planning_scene.h>
 #include <moveit/kinematic_constraints/utils.h>
-#include <moveit_msgs/DisplayTrajectory.h>
-#include <moveit_msgs/PlanningScene.h>
+#include <moveit_msgs/msg/display_trajectory.hpp>
+#include <moveit_msgs/msg/planning_scene.h>
 #include <moveit_visual_tools/moveit_visual_tools.h>
+#include <moveit/move_group_interface/move_group_interface.h>
 
 #include <boost/scoped_ptr.hpp>
```

Comment:
Can you make a quick attempt to get rid of Boost? We usually try to avoid it. I guess a unique_ptr would be fine

Reply:
Fixed.